Automattic/studio
# Studio: Remove checkboxes allowing to dismiss push and pull notifications

## Related issues

Closes https://github.com/Automattic/dotcom-forge/issues/10024

## Proposed Changes

This PR removes the `Don't ask again` checkbox on the confirmation modals for Push and Pull operations:

<img width="374" alt="Screenshot 2024-12-04 at 2 06 59 PM" src="https://github.com/user-attachments/assets/c9c0ee06-eb92-4504-8d13-cbf65047ff91">

## Testing Instructions



* Pull the changes from this branch
* Start Studio with `npm start`
* Navigate to the `Sync` tab and ensure that you have at least one site connected
* Click on the `Pull` button
* Confirm that you do not see a checkbox that allows you to dismiss notifications
* Click on the `Push` button
* Confirm that you do not see a checkbox that allows you to dismiss notifications

## Pre-merge Checklist
